pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter_test.go

```diff
@@ -213,16 +207,104 @@ var _ = Describe("Validating MigrationCreate Admitter", func() {
 				`{"very": "unknown", "spec": { "extremely": "unknown" }}`,
 				`.very in body is a forbidden property, spec.extremely in body is a forbidden property`,
 				webhooks.MigrationGroupVersionResource,
-				admitters.NewMigrationCreateAdmitter(kubevirtfake.NewSimpleClientset()).Admit,
+				admitters.NewMigrationCreateAdmitter(kubevirtfake.NewSimpleClientset(), config).Admit,
 			),
 			Entry("Migration update",
 				`{"very": "unknown", "spec": { "extremely": "unknown" }}`,
 				`.very in body is a forbidden property, spec.extremely in body is a forbidden property`,
 				webhooks.MigrationGroupVersionResource,
-				admitters.NewMigrationCreateAdmitter(kubevirtfake.NewSimpleClientset()).Admit,
+				admitters.NewMigrationCreateAdmitter(kubevirtfake.NewSimpleClientset(), config).Admit,
 			),
 		)
 	})
+
+	Context("feature gate", func() {
+		It("should reject vmim with sendTo if featuregate is disabled", func() {
```

Comment:
These 5 tests are very similar. Maybe they can be DescribeTable entries? Or at least use a common BeforeEach?